examples/bedrock-claude-summarization-accuracy.ipynb

```diff
@@ -197,17 +145,34 @@
    "id": "b252a57b",
    "metadata": {
     "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
     "pycharm": {
      "name": "#%%\n"
-    }
+    },
+    "tags": []
    },
    "outputs": [],
    "source": [
     "from fmeval.data_loaders.data_config import DataConfig\n",
     "from fmeval.model_runners.bedrock_model_runner import BedrockModelRunner\n",
-    "from fmeval.eval import get_eval_algorithm\n",
     "from fmeval.constants import MIME_TYPE_JSONLINES\n",
-    "from fmeval.eval_algorithms.factual_knowledge import FactualKnowledge, FactualKnowledgeConfig"
+    "from fmeval.eval_algorithms.summarization_accuracy import SummarizationAccuracy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3905b350-6d00-4f41-ab9f-749bea23bbd7",
+   "metadata": {},
+   "source": [
+    "#### Data Config Setup\n",
+    "\n",
+    "Below, we create a DataConfig for the local dataset file, xsum_sample.jsonl.\n",
+    "- `dataset_name` is just an identifier for your own reference\n",
+    "- `dataset_uri` is either a local path to a file or an S3 URI\n",
```

Comment:
Non-blocking question: For Clarify, `dataset_uri` is an internal-only parameter. Is this not the case for Margaret?

Reply:
`dataset_uri` is user-facing in the fmeval library.